src/Web.config

```diff
@@ -15,6 +15,10 @@
     -->
   </appSettings>
   <system.webServer>
+    <staticContent>
+      <mimeMap fileExtension="." mimeType="text/plain" />
```

Comment:
this doesn't look totally right to me. Would this allow anybody to access `https://hazzat.com/Configs/prod.json`?

Reply:
No.  Just files with no extension.  Plus express will reject the /Config routes

Reply:
Looks like the .well-known directory has its own web.config file with the mime map.  This is not needed anymore.  Removed.